x-pack/plugins/maps/server/kibana_server_services.ts

```diff
@@ -30,6 +30,7 @@ export const setIndexPatternsService = async (
   indexPatternsService = await indexPatternsServiceFactory(
     new SavedObjectsClient(getInternalRepository()),
     elasticsearchClient
+    // todo does this need to pass request
```

Comment:
Is this a question maps team needs to answer?

Reply:
@nreese Yes please

Reply:
request not needed, indexPatternsService only used with `get` and `getIds`.

Reply:
thanks!